benches/bench.rs

```diff
@@ -239,3 +241,66 @@ fn run_concurrent_round_trip<C: 'static + Client + Send + Sync>(
 	}
 	group.finish();
 }
+
+fn run_ws_concurrent_connections(rt: &TokioRuntime, crit: &mut Criterion, url: &str, name: &str, request: RequestType) {
+	let mut group = crit.benchmark_group(request.group_name(name));
+	for conns in [2, 4, 8, 16, 32, 64].iter() {
```

Comment:
If we are not supporting old rust versions you can do `for conns in [2, 4, 8, 16, 32, 64] {` :)